.babelrc

```diff
@@ -1,14 +1,22 @@
 {
-  "presets": ["./scripts/preset"],
-  "plugins": [
-    "transform-runtime",
-    "transform-flow-strip-types"
-  ],
+  "plugins": ["transform-flow-strip-types"],
   "env": {
+    "latest": {
```

Comment:
refer to the chart in this PR comment to understand env options, package.json and the build process was/is a mess. This is a lot cleaner in _my_ opinion.